tiboyce-convertsav/Makefile

```diff
@@ -0,0 +1,12 @@
+CC = gcc
+CFLAGS = -Wall -Wextra -O2
+TARGET = convertsav
+SRCS = convertsav.c lzf_c.c lzf_d.c
+HEADERS = lzf.h lzfP.h
+OBJS = $(SRCS:.c=.o)
+all: $(TARGET)
+$(TARGET): $(SRCS) $(HEADERS)
+	$(CC) $(CFLAGS) -o $@ $(SRCS)
+clean:
+	rm -f $(TARGET)
```

Comment:
You probably want to remove the OBJS files here, too (if they're being built at all? if they're not, then either add a rule for them or remove the OBJS definition)